zeppelin-interpreter/src/main/java/org/apache/zeppelin/scheduler/FIFOScheduler.java

```diff
@@ -137,15 +138,26 @@ public void run() {
               listener.jobStarted(scheduler, runningJob);
             }
             runningJob.run();
+            Object jobResult = runningJob.getReturn();
             if (runningJob.isAborted()) {
               runningJob.setStatus(Status.ABORT);
+              LOGGER.debug("Job Aborted, " + runningJob.getId() + ", " +
+                  runningJob.getErrorMessage());
+            } else if (runningJob.getException() != null) {
+              LOGGER.debug("Job Error, " + runningJob.getId() + ", " +
+                  runningJob.getReturn());
```

Comment:
shouldnt this be `runningJob.getErrorMessage()`

Reply:
LOGGER.error()?

Reply:
Job error (Paragraph error) is normal for zeppelin, so I don't think we need to make it LOGGEr.error here.

Reply:
ok